Add upper bounding for targets

djiang3099 opened the pull request:
Need to add polygon check for 'acceptable contours', oliver todo

djiang3099 committed:
Add upper bounding for targets

djiang3099 committed:
Add upper bounding for targets

raulvera59 commented on vision.py:
```diff
@@ -117,26 +119,64 @@ def find_loading_bay(self, frame: np.ndarray):
+                    cv2.circle(
+                        self.display, tuple(power_port_points[i][0]), 3, (0, 0, 255)
+                    )
+                cv2.imshow("Display", self.display)
```
Oops. This one isn't in a test, and it doesn't have a timeout. This will hang the robot.

raulvera59 reviewed (changes requested):
Found another show and waitkey, but not in test code and without a timeout. Will hang the robot.

djiang3099 committed:
Commented out imshow/waitkey

To avoid code from hanging while running on pi headless. Waitkey didn't have timeout

raulvera59 reviewed (approved):
LGTM

djiang3099 committed:
Fix merge conflicts

djiang3099 committed:
Merge branch 'master' of https://github.com/thedropbears/vision-2020 into shape_detection

djiang3099 committed:
Pulled again and ran black

Injector-Spenral commented on magic_numbers.py:
```diff
@@ -85,5 +87,11 @@
\ No newline at end of file
+ANGLE_SMOOTHING_AMOUNT = (
+    0.3  # how much of the returned angle should be the last returned one
+)
```
Why is that now a tuple?

djiang3099 committed:
Removed tuple from ANGLE_SMOOTHING
